src/components/ThemeProvider/__tests__/themeProvider-tests.jest.js

```diff
@@ -13,7 +13,7 @@ describe("ThemeProvider tests", () => {
       name: THEME_NAME
     };
     const { getByTestId } = render(
-      <ThemeProvider theme={theme} themeClassSpecifier={ADDITIONAL_STRING_SELECTOR}>
+      <ThemeProvider themeConfig={theme} themeClassSpecifier={ADDITIONAL_STRING_SELECTOR}>
```

Comment:
worth changing the `theme` variable to `themeConfig` to keep the consistency in the tests as well